codeception.yml

```diff
@@ -19,15 +19,12 @@ settings:
     # Tests (especially functional) can take a lot of memory
     # We set a high limit for them by default.
     memory_limit: 1024M
+    lint: true
 
 coverage:
     # Disable Code Coverage by default to speed up Travis tests
     enabled: false
 
-extensions:
-    enabled:
-        - Codeception\Extension\RunFailed
-
```

Comment:
Could you explain this change

Reply:
I read that since Codeception 2.1, it is "enabled by default". I assumed that meant it was enabled regardless of whether it was in the config or not. I'm now beginning to doubt myself. :wink: I'll revert it later.